nncf/experimental/openvino_native/quantization/algorithms/fast_bias_correction/ov_backend.py

```diff
@@ -0,0 +1,162 @@
+from nncf.common.tensor_statistics.collectors import ReductionShape
+from nncf.common.utils.backend import BackendType
+from nncf.common.utils.registry import Registry
+
+from nncf.experimental.openvino_native.graph.metatypes.openvino_metatypes import OV_OPERATOR_METATYPES
+from nncf.experimental.openvino_native.graph.metatypes.openvino_metatypes import OVOpMetatype
+from nncf.experimental.openvino_native.graph.metatypes.openvino_metatypes import OVConvolutionMetatype
+from nncf.experimental.openvino_native.graph.metatypes.openvino_metatypes import OVConvolutionBackpropDataMetatype
+from nncf.experimental.openvino_native.graph.metatypes.openvino_metatypes import OVMatMulMetatype
+from nncf.experimental.openvino_native.graph.metatypes.openvino_metatypes import OVFakeQuantizeMetatype
+from nncf.experimental.openvino_native.graph.metatypes.openvino_metatypes import OVConvertMetatype
+from nncf.experimental.openvino_native.graph.metatypes.openvino_metatypes import OVConstantMetatype
+from nncf.experimental.openvino_native.graph.metatypes.openvino_metatypes import OVAddMetatype
+from nncf.experimental.openvino_native.graph.transformations.commands import OVBiasCorrectionCommand
+from nncf.experimental.openvino_native.graph.transformations.commands import OVModelExtractionCommand
+from nncf.experimental.openvino_native.graph.transformations.commands import OVTargetPoint
+from nncf.experimental.openvino_native.statistics.collectors import OVMeanStatisticCollector
+from nncf.experimental.openvino_native.statistics.collectors import OVNNCFCollectorTensorProcessor
+from nncf.experimental.openvino_native.tensor import OVNNCFTensor
+from nncf.quantization.algorithms.fast_bias_correction.backend import ALGO_BACKENDS
+from nncf.quantization.algorithms.fast_bias_correction.backend import FastBiasCorrectionAlgoBackend
+
+
+@ALGO_BACKENDS.register(BackendType.OPENVINO)
+class OVFastBiasCorrectionAlgoBackend(FastBiasCorrectionAlgoBackend):
+
+    @property
+    def operation_metatypes(self) -> Registry:
+        return OV_OPERATOR_METATYPES
+
+    @property
+    def channel_axis_by_types(self) -> Dict[OVOpMetatype, int]:
+        return {
+            OVConvolutionMetatype: 1,
+            OVConvolutionBackpropDataMetatype: 1,
+            OVMatMulMetatype: -1
+        }
+
+    @property
+    def tensor_processor(self) -> OVNNCFCollectorTensorProcessor:
+        return OVNNCFCollectorTensorProcessor()
+
+    @staticmethod
+    def target_point(target_type: TargetType, target_node_name: str, port_id: int) -> OVTargetPoint:
+        return OVTargetPoint(target_type, target_node_name, port_id)
+
+    @staticmethod
+    def create_bias_correction_command(bias_node: NNCFNode, bias_value: np.ndarray) -> OVBiasCorrectionCommand:
+        bias_port_id = bias_node.layer_attributes.weight_port_id
+        target_point = OVTargetPoint(TargetType.LAYER, bias_node.node_name, bias_port_id)
+        return OVBiasCorrectionCommand(target_point, bias_value)
```

Comment:
Could you please move this method's body to the `nncf/experimental/openvino_native/graph/transformations/command_creation.py`  module as a separate method?